resources/sync_gateway_configs/xattrs/no_import_cc.json

```diff
@@ -0,0 +1,21 @@
+{
+    "interface":":4984",
+    "adminInterface": "0.0.0.0:4985",
+    "maxIncomingConnections": 0,
+    "maxCouchbaseConnections": 16,
+    "maxFileDescriptors": 90000,
+    "slowServerCallWarningThreshold": 500,
+    "compressResponses": false,
+    "log": ["*"],
+    "databases":{
+        "db":{
+            "unsupported": {
+                "enable_extended_attributes": true
```

Comment:
Don't you want to do "unsupported": {    {{ xattrs }}   } and render it at runtime?